docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: "3.7"
 
 x-env: &env
   REDIS_URL: redis://redis
+  REDIS_CACHE_URL: redis://redis
```

Comment:
That puts them both on the same redis instance, but given how rarely dev is expected to use this that's probably fine, and can be tweaked per case if it's not.